NorthstarDLL/logging.cpp

```diff
@@ -238,18 +239,29 @@ BOOL WINAPI ConsoleHandlerRoutine(DWORD eventCode)
 	return TRUE;
 }
 
-void InitialiseLogging()
+void InitialiseConsole()
 {
 	hExceptionFilter = AddVectoredExceptionHandler(TRUE, ExceptionFilter);
 
-	AllocConsole();
+	if (AllocConsole() == FALSE)
```

Comment:
Why `FALSE` and not `false`?

Reply:
or even better, just `!AllocConsole()`?

Reply:
FALSE is just defined as 0, so it really doesn't matter

I used FALSE because I'm doing winapi stuff, and also because that's how it is done in r5sdk

I can go either way on this one, i really dont mind

Reply:
Eh, I don't really mind